Examples/Algorithms/Geant4/include/ActsExamples/Geant4/ParticleKillAction.hpp

```diff
@@ -25,15 +25,16 @@ namespace ActsExamples {
 
 /// A G4SteppingAction that is called for every step in
 /// the simulation process.
-///
-/// It checks whether the particle can be killed according when its position
-/// exceeds the configured values for |z| or r.
 class ParticleKillAction : public G4UserSteppingAction {
  public:
   /// Configuration of the Stepping action
   struct Config {
+    /// particles outside this volume will be terminated
     std::shared_ptr<const Acts::Volume> volume;
```

Comment:
Is the possibility to have ownership of the volume here just to have the ability to define a custom volume?

Reply:
yes initially the idea was to pass an arbitrary volume here. by default we pass the detector volume now